Composer/packages/tools/language-servers/intellisense/src/intellisenseServer.ts

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  CompletionParams,
+  DidChangeConfigurationParams,
+  IConnection,
+  InitializeParams,
+  TextDocuments,
+} from 'vscode-languageserver';
+import { CompletionItem, CompletionList } from 'vscode-languageserver-types';
+import { expressionsResolver } from './resolvers/expressions';
+import { userVariablesResolver } from './resolvers/userVariables';
+import { variableScopesResolver } from './resolvers/variableScopes';
+import { getCompletionString, getRangeAtPosition } from './utils/intellisenseServerUtils';
+const Fuse = require('fuse.js');
```

Comment:
Why is this not imported?

Reply:
There is a namespace conflict error...

Reply:
Can you expand on this?

Reply:
Soroush helped me fix this. "esModuleInterop" needed to be true.